### PR TITLE
Mk qa cleanup

### PR DESCRIPTION
**Pull request by markkramerus**
Worked on slicing, got rid of another batch of errors. Not perfect, but progress and a good stopping point to get into master.

### CODE REVIEW

File: CancerGenomics.fsh [hunk 1]
```diff
@@ -31,14 +31,9 @@ Description:    "Records an alteration in the most common DNA nucleotide sequenc
     GenomicDNAChange 0..1 MS and
     GenomicSourceClass 0..1 MS and
     AminoAcidChange 0..1 MS and
-    AminoAcidChangeType 0..1 MS and
+    AAChangeType 0..1 MS and
```
Comment (mlterryMitre-zz) [inline CancerGenomics.fsh]: Let's not change this. AminoAcidChangeType makes it easier to align with the component name as specified in the CG Reporting IG.  It also let's the implementer know that it is associated with the previous component AminoAcidChange.

**Review by mlterryMitre-zz — CHANGES_REQUESTED** [review]
@markkramerus - Revert any naming from AAChangeType back to AminoAcidChangeType.  This is done on purpose to be explicit and more easily readable to align with the CG Reporting IG.
Shortening this makes it more ambiguous to all implementers.

**Comment by cmoesel** [comment]
@mlterryMitre -- I think this is actually a short-term workaround due to bug FHIR/sushi#122.  The slice generation is currently broken if one slice name is a substring of another -- so the combination of `AminoAcidChange` and `AminoAcidChangeType` is causing problems.  This can be reverted back once the bug is fixed in SUSHI.  But... perhaps a comment in the FSH would be helpful so other authors understand this and so we remember to change it back later.

**Comment by mlterryMitre-zz** [comment]
Thanks for clarifying @cmoesel. Any idea when FHIR/sushi#122 will be fixed?  I really prefer not to publish mCODE PreRelease#2 with this component name and changing it after affected reviewers in the CGWG who commented on this have withdrawn negative votes. Because the changes from @markkramerus were broad (including the affected examples), I would prefer that we just comment out the component for this PR and uncomment it when it's fixed before the PreRel#2 release.

File: CancerGenomics.fsh (hunk 1 above)
Comment (markkramerus) [reply CancerGenomics.fsh]: That's not the reason for this change. This is a workaround for a SUSHI bug https://github.com/FHIR/sushi/issues/122. We will change the name back (as noted in comments) when the bug is fixed.

Comment (mlterryMitre-zz) [reply CancerGenomics.fsh]: when will the bug be fixed?  I strongly prefer not to release PreRel#2 with this component name.

**Review by mlterryMitre-zz — APPROVED** [review]

**Pull request merged**